.github/workflows/main.yml

```diff
@@ -0,0 +1,32 @@
+name: Services
+on:
+  push:
+    branches: [main]
+
+jobs:
+  build_deploy:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: ./frontend
+    steps:
+      - name: checkout
+        uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
+        with:
+          node-version: 16.14.2
+      - name: Installing Dependencies
+        run: npm install
+      - name: Building
+        run: npm ci
+      - name: Deploying
+        uses: SamKirkland/FTP-Deploy-Action@v4.3.4
+        with:
+          server: ${{ secrets.FTP_SERVER }}
+          username: ${{ secrets.FTP_USERNAME }}
+          password: ${{ secrets.FTP_PASSWORD }}
+          local-dir: "frontend/build/"
+          exclude: |
+            **/.git*
+            **/.git*/**
+            **/node_modules/**
```

Comment:
Since we are copying from `frontend/build/` folder, do we need this `exclude` parameter? We are anyway going to copy everything from `frontend/build/`.

Reply:
No, we don't require the exclude parameter now. I have removed it from the files.